repo/blob/s3/s3_storage.go

```diff
@@ -408,12 +450,6 @@ func newStorageWithCredentials(ctx context.Context, creds *credentials.Credentia
 		return nil, errors.Errorf("bucket %q does not exist", opt.BucketName)
 	}
```

Comment:
Move these 2 error checks to the helper function and only return error from the helper, as mentioned above.

Reply:
Done